src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -205,7 +206,7 @@ private void addConcert(int start, int anz) {
                                 while ((p1 = seite2.indexOf(a, p1)) != -1) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)